lib/views/quiz/quiz_index.dart

```diff
@@ -25,10 +30,30 @@ class _QuizIndexState extends State<QuizIndex> {
 
   @override
   Widget build(BuildContext context) {
+    Screen.width(context) >= 600
+        ? handleOrientation([
+            DeviceOrientation.portraitUp,
+            DeviceOrientation.landscapeLeft,
+            DeviceOrientation.landscapeRight
+          ])
+        : handleOrientation([DeviceOrientation.portraitUp]);
+
     return Scaffold(
       backgroundColor: Theme.of(context).backgroundColor,
       body: quizzes.length > 0
-          ? ListView(children: buildCards(context, quizzes))
+          ? GridView.count(
+              mainAxisSpacing: Screen.percentOfWidth(context, 97.5),
+              crossAxisSpacing: Screen.percentOfWidth(context, 97.5),
+              crossAxisCount: (Screen.width(context) >= 600 &&
```

Comment:
If you implement this in Screen:
`bool isPortrait() => orientation == Orientation.portrait;`

This can be simplified to:
```
Screen.width(context) >= 600 ?
    Screen.isPortrait() ? 3 : 2 :
    : 1;
```

Reply:
Implemented